### ISSUE TITLE
Package is not working anymore: cannot find module '@cedx/akismet'

### ISSUE THREAD

**Issue by cedx** [cedx/akismet#5]
It seems that the NPM `require` resolution does not find this module...

```
Error: Cannot find module '@cedx/akismet'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at repl:1:17
    at sigintHandlersWrap (vm.js:22:35)
    at sigintHandlersWrap (vm.js:96:12)
    at ContextifyScript.Script.runInThisContext (vm.js:21:12)
    at REPLServer.defaultEval (repl.js:313:29)
    at bound (domain.js:280:14)
```

**Comment by cedx** [comment]
Confirmed: the `lib` folder containing the source code is not packaged into the NPM release, due to the absence of a `.npmignore` file...
The NPM publishing used the `.gitignore` file... which ignore the `lib` folder 😞

**Comment by cedx** [comment]
Fixed in version 1.0.1.
